app/commands/v2/put_content.rb

```diff
@@ -98,6 +99,12 @@ def create_links(edition)
         end
       end
 
+      def fetch_embedded_content(edition)
+        return {} if edition[:details]["body"].nil?
```

Comment:
We should think about whether we want to support things other than `body` - in particular `parts` in multi part content will probably need support for embeds.